code/modules/overmap/objects/jump_point.dm

```diff
@@ -0,0 +1,76 @@
+/**
+ * # Jump points
+ *
+ * These overmap objects can be interacted with and will send you to another sector.
+ * These will then dump you onto another jump point post jump. Useful for events or punchcards!
+ */
+/datum/overmap/jump_point
+	name = "jump point"
+	desc = "A specific point in this system where you are able to bluespace 'jump' to one specific system."
+	char_rep = "~"
+	token_icon_state = "jump_point"
+
+	interaction_options = list(INTERACTION_OVERMAP_JUMPTO)
+
+	///Direction we are facing, used for token mostly
+	var/dir
+	///The currently linked jump point
+	var/datum/overmap/jump_point/destination
+
+
+/datum/overmap/jump_point/Initialize(position, datum/overmap_star_system/system_spawned_in, _other_point, ...)
+	. = ..()
+	if(_other_point)
+		link_points(_other_point)
+
+/datum/overmap/jump_point/proc/link_points(other_point)
```

Comment:
this really needs better feedback for admin usage. It shouldn't be a pain in the ass to create jump points and link them.